importer/BeBruNl.py

```diff
@@ -10,14 +10,23 @@ def set_adm_location(self):
         location = self.plaats
         municip_dict = self.data_files["municipalities"]
         if location:
-            location_match = [x for x in municip_dict if x[
-                "itemLabel"] == location]
+            location_match = [x for x in municip_dict
```

Comment:
Maybe als obreak this out as a util function since the basic structure holds for all your data files.

```python
def get_matching_items_from_data_file(data_list, value, value_field="itemLabel", target_field="item"):
    return [x[target_field] for x in data_list if x[value_field] == value]
```

Reply:
Done, added to utils w/tests.

Reply:
See question in chat